containers/kubernetes/api-cli/managing-storage.mdx

```diff
@@ -18,175 +18,69 @@ categories:
-    ```bash
-    kubectl apply -f importing/pod.yaml
-    ```
-
-## Encrypting volumes
-
-This plugin supports at rest encryption of the volumes with Cryptsetup/LUKS.
+The output of this command provides a quick status update on the CSI plugin within your Kubernetes cluster. For the latest features and enhancements, consider upgrading to [release 0.3](https://github.com/scaleway/scaleway-csi/tree/release-0.3#block-storage-low-latency), which supports **[Block Storage low latency](/storage/block/quickstart/)** volumes.
+
+To identify your current CSI release version, navigate to the [Cockpit interface](/observability/cockpit/how-to/access-grafana-and-managed-dashboards/), specifically the **Kubernetes Cluster - Overview** dashboard.
 
 <Message type="note">
-  Resizing an encrypted volume [does not work](https://github.com/container-storage-interface/spec/issues/445).
+   * In February 2024, block volumes and snapshots management will transition to the [Scaleway Block Storage API](https://www.scaleway.com/en/developers/api/block/). This means, you can view your block volumes in the [Block Storage](https://console.scaleway.com/block-storage/volumes) section of the Scaleway console.
+   * Anticipate the availability of the **boot on block** feature for low-latency Block Storage volumes in Q2.
 </Message>
 
-### Storage class parameters
+## Upgrading to CSI version 0.3
 
-In order to have an encrypted volume, `encrypted: true` needs to be added to the `StorageClass` parameters.
-You will also need a passphrase to encrypt/decrypt the volume, which is taken from the secrets passed to the `NodeStageVolume` method.
+1. Initiate the migration process by adding the following tag to your cluster:
+   ```bash
+   enable-csi-v0.3=migrate
+   ```
+2. After updating the cluster tag, the cluster switches to the _updating_ status, indicating the deployment of the new CSI v0.3 on your nodes.
 
-The [external-provisioner](https://github.com/kubernetes-csi/external-provisioner) can be used to [pass down the wanted secret to the CSI plugin](https://kubernetes-csi.github.io/docs/secrets-and-credentials-storage-class.html) (v1.0.1+).
+    <Message type="important">
+        Once the migration to CSI v0.3 is initiated, a **rollback to CSI v0.2 is impossible**.
+    </Message>
 
-Two additional parameters are needed on the StorageClass:
-- `csi.storage.k8s.io/node-stage-secret-name`: The name of the secret
-- `csi.storage.k8s.io/node-stage-secret-namespace`: The namespace of the secret
+    <Message type="note">
+      Note: This migration process applies to PVCs (Persistent Volume Claims) and snapshots only. System volumes remain on the Instances side, unaffected by this migration.
+    </Message>
 
-The secret needs to have the passphrase in the entry with the key `encryptionPassphrase`.
+3. Upon completion of the migration process, verify the successful transfer of PVCs and snapshots to the [SBS-API]((https://www.scaleway.com/en/developers/api/block/)). 
```

Comment:
Should the link be in between two (( )) ?

Reply:
```suggestion
3. Upon completion of the migration process, verify the successful transfer of PVCs and snapshots to the [SBS-API](https://www.scaleway.com/en/developers/api/block/). 
```